### PR TITLE
update 0610 and 1721 ACS data for 2022 build

### PR DESCRIPTION
**Pull request by damonmcc**
## changes
- many changes are related to previous manual data update for 2020
- replace use of `poetry` process with `requirements.txt`
- generalize update action/script so it isn't limited to 2020
- add data for 2017 - 2021 (aka 2021) ACS data for the manual update
- simplify dev container
- fix `test_metadata.py`
- update `.gitignore` and remove ad-hoc `.gitignore` approaches in __init__ files
- add `example.env` file

## notes
- [Manual update run](https://github.com/NYCPlanning/db-factfinder/actions/runs/4307452790/jobs/7512537401) that generated 2021 ACS data
  - compared several variables in output to those in  Population Team's excel file and they align
- [Manual update run](https://github.com/NYCPlanning/db-factfinder/actions/runs/4318592999) that generated new 2010 ACS data
  - compared to old data in `edm-publishing` and new values are 1.2542905 higher than old values
- many changes here are from Andrew, Jingyi, and Sasha and were used to generate 2020 data. the last commit from their work was on August 3, 2022
- previous 2010 data has been archived in `edm-publishing` by placing it in a `3-1-22` folder next to the new output. there was already a `10-6-21` folder

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -5,51 +5,22 @@
...
-	],
-
-	// Uncomment the next line to run commands after the container is created - for example installing curl.
-	"postCreateCommand": ". $HOME/.poetry/env && poetry install",
-
 	// Adding id_rsa so that we can push to github from the dev container
 	"initializeCommand": "ssh-add $HOME/.ssh/id_rsa",
-	
+	// Uncomment the next line to run commands after the container is created - for example installing curl.
+	"postCreateCommand": "./bash/dev_python_packages.sh",
```
Comment (mbh329) [inline .devcontainer/devcontainer.json:19]: Nice - I like this improvement

File: pipelines/acs_manual_update.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
+import argparse
```
Comment (mbh329) [inline pipelines/acs_manual_update.py:1]: A general description of the "why" might be nice here. "We recieve data directly from DCP Population in an excel file that necessitates cleaning the data to a standard form OSE uses to ingest the data into Population FactFinder. This script has functions that handle standardizes column schema, etc..."

Comment (damonmcc) [reply pipelines/acs_manual_update.py:2]: see 99fc971

**Review by mbh329 — APPROVED** [review]
Looks good to me.

**Comment by damonmcc** [comment]
cc @fvankrieken for visibility on a recent not-small PR

**Pull request merged**